src/test/rpc_tests.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include <univalue.h>
 
+#include <rpc/blockchain.cpp>
```

Comment:
Can't you just include the .h file?

Reply:
If there are static methods that you need, remove the static in the cpp file and add the declaration here

Reply:
Yup, that works thanks. I had to move some of the other includes but I think that should be okay.